### ISSUE TITLE
Data init API for TFLite Swift

### ISSUE THREAD

**Issue by drubinstein** [tensorflow/tensorflow#56144]
<details><summary>Click to expand!</summary> 
 
 ### Issue Type

Feature Request

### Source

source

### Tensorflow Version

2.8+

### Custom Code

No

### Current Behaviour?

```shell
The current Swift API only has `init` functions from files on disk unlike the Java (Android) API which has a byte buffer initializer. It'd be convenient if the Swift API could initialize `Interpreters` from `Data`.
```

### Standalone code to reproduce the issue

```shell
No code. This is a feature request
```

### Relevant log output

_No response_</details>

**Comment by mohantym** [comment]
Hi @drubinstein !This issue will be closed once this [PR ](https://github.com/tensorflow/tensorflow/pull/56145) is merged. Thank you!

**Comment by pjpratik** [comment]
Hi @drubinstein 

The PR #56145 has been merged with commit eeae14f45e7df18868969dcbc678ad49295caaf9

Could you please confirm and close the issue if it is resolved?

Thanks.
